packages/tables/src/Grouping/Group.php

```diff
@@ -250,7 +250,7 @@ public function getKey(Model $record): mixed
         return Arr::get($record, $this->getColumn());
     }
 
-    public function getTitle(Model $record): string | Htmlable
+    public function getTitle(Model $record): ?(string | Htmlable)
```

Comment:
This is not valid PHP. It is very important to test your changes locally before you PR them. Have you tested that this is the only place that needs to be updated in order to support null groups?

```suggestion
    public function getTitle(Model $record): string | Htmlable | null
```

Reply:
@philmareu I changed the PR now and hope to get it accepted :)